components/insight/SRC/org/openmicroscopy/shoola/agents/measurement/util/roitable/ROINode.java

```diff
@@ -216,6 +229,19 @@ public ROINode findChild(Coord3D shapeCoord)
 		return null;
 	}
 	
+    /**
+     * Find the child node representing the given folder
+     * 
+     * @param folder
+     *            see above.
+     * @return see above.
+     */
+    public ROINode findChild(FolderData folder) {
+        if (folderMap.containsKey(folder))
+            return folderMap.get(folder);
```

Comment:
That's true, thanks. Was just copy/pasting the other findChild() method, without really thinking about it.